### PR TITLE
Data Cleanup - Fixes Store Name Casing Issue

### PR DESCRIPTION
**Pull request by SirWellington**

### CODE REVIEW

**Comment by SirWellington** [comment]
Can one of the admins verify this patch?

**Comment by cordhernandez** [comment]
Verified.

File: src/main/java/tech/blacksource/blacknectar/service/data/cleanup/RunStoreTransformation.java [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+import tech.blacksource.blacknectar.service.stores.Store;
+
+import static tech.sirwellington.alchemy.arguments.Arguments.checkThat;
+import static tech.sirwellington.alchemy.arguments.assertions.Assertions.notNull;
+
+/**
+ *
+ * @author SirWellington
+ */
+public class RunStoreTransformation implements Callable<Void>
```
Comment (SirWellington) [inline src/main/java/tech/blacksource/blacknectar/service/data/cleanup/RunStoreTransformation.java:38]: Add a unit test for this now that it's separated out

Comment (SirWellington) [reply src/main/java/tech/blacksource/blacknectar/service/data/cleanup/RunStoreTransformation.java:38]: Done!

**Pull request merged**